tests/std/tests/P0896R4_ranges_iterator_machinery/test.cpp

```diff
@@ -1896,11 +1866,14 @@ namespace iter_ops {
         constexpr explicit trace_iterator(int const pos, trace& t) noexcept(NoThrow == nothrow::yes)
             : trace_{&t}, pos_{pos} {}
 
-        trace_iterator(trace_iterator const&) requires is_forward = default;
-        trace_iterator(trace_iterator&&)                          = default;
+        trace_iterator(trace_iterator const&)
+            requires is_forward
+        = default;

```

Comment:
Ditto, "wrapped defaulted functions are too weird for me" - I'm going to audit for these.